working/augmentations/feature-specification.md

```diff
@@ -803,17 +794,27 @@ augment class C {
 }
 ```
 
-An optional formal parameter has the default value _d_ if exactly one
+An optional formal parameter has the default value *d* if exactly one
 declaration of that formal parameter in the augmentation chain specifies a
-default value, and it is _d_. An optional formal parameter does not have an
+default value, and it is *d*. An optional formal parameter does not have an
 explicitly specified default value if none of its declarations in the
 augmentation chain specifies a default value. The default value is
-introduced implicitly with the value null in the case where the parameter
+introduced implicitly with the value `null` in the case where the parameter
 has a nullable declared type, and no default values for that parameter are
 specified in the augmentation chain.
 
 It's a **compile-time** error if:
 
+*   An augmentation declaration is applied to a declaration of a different kind.
+    For example, augmenting a function with a `class`, a method with a getter, a
+    constructor with a static method, etc.
```

Comment:
Repeats line 662, which said the same thing for class-like augmentations.
Since this error goes across that dinstinction, the kinds can be either, maybe specify it before the "class-like declarations" and "function-declarations" split.

Then, inside the two parts, you don't have to worry about whether it's the introducing or the augmenting declaration which is the class-like/funciton declaration.